stable/cockroachdb/templates/cockroachdb-statefulset.yaml

```diff
@@ -236,7 +240,7 @@ spec:
       {{- if .Values.NodeSelector }}
       nodeSelector:
 {{ toYaml .Values.NodeSelector | indent 8 }}
-      {{- end }}
+{{- end }}
```

Comment:
Why change this? It probably doesn't matter, but having the `if` and the `end` indented differently scares me.